downstream/modules/navigator/ref-faq-navigator.adoc

```diff
@@ -8,11 +8,11 @@ Use the following {Navigator} FAQ to help you troubleshoot problems in your envi
 
 Where should the `ansible.cfg` file go when using an {ExecEnvNameSing}?:: The easiest place to have the `ansible.cfg` is in the project directory adjacent to the playbook. The playbook directory is automatically mounted in the execution environment and the `ansible.cfg` file will be found.  If the `ansible.cfg` file is in another directory, the `ANSIBLE_CONFIG` variable needs to be set and the directory specified as a custom volume mount. (See {Navigator} settings for `execution-environment-volume-mounts`)
 +
-Where should the `ansible.cfg` file go when not using an {ExecEnvNameSing} Ansible looks for the `ansible.cfg` in the typical locations when not using an {ExecEnvNameSing}. See https://docs.ansible.com/ansible/latest/reference_appendices/config.html[Ansible configuration settings] for details.
+Where should the `ansible.cfg` file go when not using an {ExecEnvNameSing} Ansible looks for the `ansible.cfg` in the typical locations when not using an {ExecEnvNameSing}. See link:https://docs.ansible.com/ansible/latest/reference_appendices/config.html[Ansible configuration settings] for details.
 
 Where should Ansible collections be placed when using an {ExecEnvNameSing}?:: The easiest place to have Ansible collections is in the project directory, in a playbook adjacent collections directory (for example, `ansible-galaxy collections install ansible.utils -p ./collections`).  The playbook directory is automatically mounted in the {ExecEnvNameSing} and {Navigator} finds the collections there. Another option is to build the collections into an {ExecEnvNameSing} using {Builder}. This helps content creators author playbooks that are production ready, since {ControllerName} supports playbook adjacent collection directories. If the collections are in another directory, set the `ANSIBLE_COLLECTIONS_PATHS` variable  and configure a custom volume mount for the directory. (See xref:ref-navigator-general-settings_settings-navigator[Automation content navigator general settings] for `execution-environment-volume-mounts`).
 
-Where should ansible collections be placed when not using an {ExecEnvNameSing}?:: When not using an {ExecEnvNameSing}, Ansible looks in the default locations for collections. See the https://docs.ansible.com/ansible/latest/user_guide/collections_using.html[Ansible Collections User Guide].
+Where should ansible collections be placed when not using an {ExecEnvNameSing}?:: When not using an {ExecEnvNameSing}, Ansible looks in the default locations for collections. See the link:* link:https://docs.ansible.com/ansible/latest/collections_guide/index.html[Using Ansible collections] guide.
```

Comment:
should the words "the link" and the asterisk be there? I noticed it because that's not how it's formatted in line 11--should these be consistent?

Reply:
@hherbly 
I removed "link:*"
Line 11 reads as "See Ansible configuration settings for details"
Line 15 renders as "See the Using Ansible Collections guide". - so I thought I needed to keep "the"

For the moment, I've kept the "the" in line 15, but I can change it.

Reply:
Oh yes that does make sense. No need to remove. Thanks!